packages/admin/admin/src/stack/breadcrumbs/BreadcrumbsEntry.tsx

```diff
@@ -1,42 +1,80 @@
 import { LevelUp } from "@comet/admin-icons";
-import { IconButton, Link, Typography } from "@mui/material";
-import { WithStyles } from "@mui/styles";
-import clsx from "clsx";
+import { IconButton as MuiIconButton, Link, Typography } from "@mui/material";
+import { css, styled } from "@mui/material/styles";
 import * as React from "react";
 
 import { BreadcrumbItem } from "../Stack";
 import { BreadcrumbLink } from "./BreadcrumbLink";
-import { styles } from "./StackBreadcrumbs.styles";
+import { BackButtonSeparator, StackBreadcrumbsProps } from "./StackBreadcrumbs";
+
+const StyledLink = styled(Link, {
```

Comment:
I'd prefer if you'd remove the Styled prefixes... They don't really add any meaning to the components. For the name clash on Link you could import the Link as `Link as MuiLink`